.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
Not related to your changes:

I must admit I *despise* how VSCode just doesn't respect the JSON spec. This file should absolutely have a `.jsonc` extension to allow comments.

Reply:
ha! I'll purge these :)

Reply:
TIL: I didn't realize there's a JSONC proposal there. @jack-r-warren